Homework 6 contains 
promises.js
Contains function that sends a request using fetch to retrieve a JSON resource and processes it with another function 
using .then()

try-catch.js sends to invalid url request and gets error after that sends to the next url thats valid.